src/test/java/org/jenkinsci/plugins/stashNotifier/StashNotifierTest.java

```diff
@@ -111,4 +127,59 @@ public void test_prebuild_null_revision() {
 		when(build.getActions(BuildData.class)).thenReturn(Collections.singletonList(mock(BuildData.class)));
 		assertTrue(sn.prebuild(build, buildListener));
 	}
+
+    @Test
+    public void test_build_http_client_with_proxy() throws Exception {
+        //given
+        String address = "109.196.210.110";
```

Comment:
Not sure we should use any public IP even for mock data. Replace with private network IP please